docs/Extensibility_UG/add_openvino_ops.md

```diff
@@ -0,0 +1,62 @@
+# Custom OpenVINO™ Operations {#openvino_docs_Extensibility_UG_add_openvino_ops}
+
+OpenVINO™ Extension API allows you to register custom operations to support models with operations which OpenVINO™ does not support out-of-the-box.
```

Comment:
allows to create and register.

What if I want to map unknown operation to existing OV? Guide does not show me in what cases I need to write custom operation, in what cases I can reuse operations from OV opset.

Reply:
We also will have some documentation about OpConversion extensions. But it will be added separately